rails/app/controllers/developments_controller.rb

```diff
@@ -64,7 +64,10 @@ def create
   # PATCH/PUT /developments/1
   def update
     authorize @development
-    if @development.update(development_params)
+
+    validate = !development_params[:flag]
+    @development.attributes = development_params
+    if @development.save(validate: validate)
```

Comment:
Why would we not want to validate if a development has been flagged?

Reply:
I don't fully remember the logic but I remember there being a reason for it. @mzagaja wrote this, so he might know better.

Reply:
I don't fully remember the logic but I remember there being a reason for it. @mzagaja wrote this, so he might know better.

Reply:
When you update a flag boolean on a development it runs validations but a large number of developments do not currently pass validations. So if you try to flag or unflag a development and do not skip validations, then it will not let you change its status.

Reply:
As long as an anonymous user can't edit other properties when they set a flag, that works.

Reply:
Not from the interface. If someone were to look at the request bodies and take a shot in the dark with the validations by sending out a request where flag is set along with other attributes.